client/iaid.c

```diff
@@ -92,19 +151,28 @@ ni_do_iaid_dump(int argc, char **argv)
 static int
 ni_do_iaid_get(int argc, char **argv)
 {
-	enum {	OPT_HELP = 'h' };
+	enum {	OPT_HELP = 'h', OPT_IAID_FMT = 1 };
 	static struct option    options[] = {
-		{ "help",	no_argument,		NULL,	OPT_HELP	},
-		{ NULL,		no_argument,		NULL,	0		}
+		{ "help",		no_argument,		NULL,	OPT_HELP	},
+		{ "iaid-format",	required_argument,	NULL,	OPT_IAID_FMT	},
+		{ NULL,			no_argument,		NULL,	0		}
 	};
 	int opt = 0, status = NI_WICKED_RC_USAGE;
 	ni_iaid_map_t *map = NULL;
 	const char *ifname = NULL;
 	unsigned int iaid;
+	ni_iaid_fmt_t opt_iaid_fmt = OPT_IAID_FMT_DEC;
 
 	optind = 1;
-	while ((opt = getopt_long(argc, argv, "+h", options, NULL)) != EOF) {
+	while ((opt = getopt_long(argc, argv, "+hx", options, NULL)) != EOF) {
```

Comment:
```suggestion
	while ((opt = getopt_long(argc, argv, "+h", options, NULL)) != EOF) {
```
I don't see the reason for this `x`?

Reply:
It was an leftover from another variant.